### DISCUSSION TITLE
Connecting to Azure IoT Hub using MQTT client

### DISCUSSION THREAD

**Discussion by sivapriyanc** [adam-fowler/mqtt-nio#152] [Q&A]
Hi All, 
I am trying to connect to Azure IoT Hub using mqttnio, followed the sample which has been mentioned Python sample with PahoMQTT But not able to connect  with device certificate(private key and x509) combination . these options are working with Paho for the same credentials, any help is much appreciated

**Comment by adam-fowler** [comment]
Can you post your code so we can see what you are doing. Also what errors do you get back?
